lib/SILGen/SILGen.cpp

```diff
@@ -1277,8 +1277,8 @@ emitMarkFunctionEscapeForTopLevelCodeGlobals(SILLocation loc,
 }
 
 void SILGenModule::emitAbstractFuncDecl(AbstractFunctionDecl *AFD) {
-  // Emit any default argument generators.
-  emitDefaultArgGenerators(AFD, AFD->getParameters());
+  // Emit any argument generators.
```

Comment:
Nitpick: Should this say "default arguments and property wrapper initializers"? Otherwise I feel like the comment doesn't add much value since the function name is already clear.